monitoring/supervisord.conf

```diff
@@ -0,0 +1,12 @@
+[supervisord]
+nodaemon=true
+
+[program:prometheus]
```

Comment:
Check this example for logging — the `/dev/fd/1` trick here seems to work well for getting stdout on the container. https://github.com/livepeer/livepeer-wowza/blob/master/etc/WowzaStreamingEngine.conf